DigitalLearningSolutions.Data/DataServices/UserDataService/AdminFieldsDataService.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace DigitalLearningSolutions.Data.DataServices.UserDataService
+{
+    using System.Linq;
+    using Dapper;
+
+    public partial class UserDataService
+    {
+        public int GetAnswerCountForCourseAdminField(int customisationId, int promptNumber)
+        {
+            return connection.Query<string>(
+                $@"SELECT Answer{promptNumber}
+                        FROM Progress
+                        WHERE CustomisationID = @customisationId AND Answer{promptNumber} IS NOT NULL",
+                new { customisationId }
+            ).Count(x => !string.IsNullOrWhiteSpace(x));
+        }
+
+        public void DeleteAllAnswersForAdminField(int customisationId, int promptNumber)
```

Comment:
Re naming - could we call this "ForCourseAdminField" too, like above :)

Reply:
done :)